src/fides/api/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -2003,6 +2021,8 @@ def create_privacy_request_func(
             "Application redis cache required, but it is currently disabled! Please update your application configuration to enable integration with a redis cache."
         )
 
+    privacy_preferences = privacy_preferences or []
```

Comment:
I changed the way I did this check so my response to your old comment got flagged as "outdated" but I wanted to make sure you read my response.

---
I set this as a `None` value to avoid this gotcha with mutable default arguments https://docs.python-guide.org/writing/gotchas/. We don't mutate `privacy_preferences` in our function right now but I figured I'd do this now to avoid any pitfalls in the future.

I also changed the `None` check to this to let mypy know that `privacy_preferences` will be valued, even if it's an empty array
```
privacy_preferences = privacy_preferences or []
```

Reply:
Oh noooo this is the worst gotcha! Wow, thanks for explaining!